docs/upgrade-guides/3-11-to-3-12.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Upgrading from 3.11.x to 3.12
+---
+
+## Preparation
+
+Version 3.12 introduces new fields in our model. To properly migrate from to 3.12 please migrate first to version 3.11.4
+
+## Reasoning
+
+In version 3.12 `orderSettings` were moved to `Channel`. For more info relate to https://github.com/saleor/saleor/blob/main/CHANGELOG.md
```

Comment:
```suggestion
For more info relate to https://github.com/saleor/saleor/blob/main/CHANGELOG.md
```

Three points here:
- please make sure to include the "." at the end of the sentence
- please format the links, e.g. `[Changelog](https://github.com/saleor/saleor/blob/main/CHANGELOG.md)`
- the sentence should probably be something like "for more related information, please...", but...

...I think the documentation should be the source of truth here. Changelog is a nice addition, but we should at least briefly inform about the consequences of the appointed changes.